composer.json

```diff
@@ -16,7 +16,7 @@
     "spryker/testify": "*"
   },
   "autoload": {
-    "psr-0": {
+    "psr-4": {
```

Comment:
That is not how psr-4 works!

Reply:
I've already known it. I should change my branches quickly and I couldn't find the right way earlier